pyccel/codegen/printing/ccode.py

```diff
@@ -8,19 +8,24 @@
 from pyccel.ast.core import SeparatorComment, VariableAddress
 from pyccel.ast.core import DottedName
 
-from pyccel.ast.core import PyccelAdd, PyccelMul, String
+from pyccel.ast.core import Declare, IndexedVariable, Slice
 
+from pyccel.ast.core import PyccelAdd, PyccelMul, String, PyccelMinus
+from pyccel.ast.core      import PyccelUnarySub, PyccelMod
 from pyccel.ast.datatypes import default_precision
 from pyccel.ast.datatypes import NativeInteger, NativeBool, NativeComplex, NativeReal, NativeTuple
 
 
 from pyccel.ast.numpyext import NumpyFloat
 from pyccel.ast.numpyext import NumpyReal, NumpyImag
+from pyccel.ast.numpyext import NumpyFull, NumpyArray
+from pyccel.ast.numpyext import Shape
+from pyccel.ast.numpyext import Tuple
```

Comment:
Why are we importing `Tuple` from `pyccel.ast.numpyext` rather than directly from sympy?

Reply:
is there a big difference with the 2 cases?

Reply:
`Tuple` is defined in sympy. You can import it from `pyccel.ast.numpyext` because that file defines at this line:
https://github.com/pyccel/pyccel/blob/master/pyccel/ast/numpyext.py#L9
```
from sympy import Tuple
```
Technically there is no difference as you are importing the same object either way, but not importing it from where it is defined adds a step when trying to find the definition and dupes us into thinking that the `CCodePrinter` doesn't depend on sympy when it apparently does. We are slowly removing the sympy dependencies so hiding one is really unhelpful